maps/__DeepTunnels/map/_Nadezhda_Deep_Tunnels.dmm

```diff
@@ -8745,11 +8777,18 @@
 	name = "murky water"
 	},
 /area/nadezhda/rnd/mixing)
+"yc" = (
+/obj/machinery/atmospherics/unary/vent_scrubber/on{
```

Comment:
We dont use active vents for sake of lag

Reply:
Fixed, thank you and CDB for pointing that one